.travis.yml

```diff
@@ -161,7 +161,7 @@ script:
 
         (make check-unit-coverage \
          && do_codecov unittests \
-         && sed "1 s~^~require('luacov.runner')('$PWD/.luacov'); \0~" build/awesomerc.lua \
+         && sed -i "1 s~^~require('luacov.runner')('$PWD/.luacov'); \0~" build/awesomerc.lua \
```

Comment:
The "original line numbers" would require to use `../awesomerc.lua`. `build/awesomerc.lua` have been modified by the build system to remove the section markers used when generating the documentation.